tools/dependency/release_dates.py

```diff
@@ -38,35 +40,69 @@ def format_utc_date(date):
 def verify_and_print_latest_release(dep, repo, metadata_version, release_date):
     try:
         latest_release = repo.get_latest_release()
-        if latest_release.created_at > release_date and latest_release.tag_name != metadata_version:
-            print(
-                f'*WARNING* {dep} has a newer release than {metadata_version}@<{release_date}>: '
-                f'{latest_release.tag_name}@<{latest_release.created_at}>')
-    except github.UnknownObjectException:
-        pass
+    except github.GithubException as e:
+        print('GithubException {e} while getting latest release.')
+        return
+    if latest_release.created_at > release_date and latest_release.tag_name != metadata_version:
+        print(
+            Fore.YELLOW
+            + f'*WARNING* {dep} has a newer release than {metadata_version}@<{release_date}>: '
```

Comment:
nit: unnecessary `+`

Reply:
Moved all `Fore.` and `Style.` into f strings and removed extraneous `+`